### PR TITLE
Assignment1 @schuywalker

### PR DESCRIPTION
**Pull request by tremorris1999**

### CODE REVIEW

**Review by linhtran8699 — COMMENTED** [review]
The website is up and running. Great job done!

File: wordle-ad-farm/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # wordle-ad-farm
 ## Azure Deployment
 You can view the deployed project [here](https://polite-desert-067c8041e.1.azurestaticapps.net).
+https://polite-desert-067c8041e.1.azurestaticapps.net/
```
Comment (ZackWard-student) [inline wordle-ad-farm/README.md]: Would have been nice to see this as a comment as well. Love the random name polite-desert!

File: wordle-ad-farm/pages/game.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<template>
+    
+    <v-card class='content-vue' >
+        
+        <v-card v-if="!isLoading" loading >
+            <!-- pre view -->
+            
+            <!-- <v-row > -->
+            
+            <p>you're being exploited for ad revenue. please wait a moment...</p>
```
Comment (ZackWard-student) [inline wordle-ad-farm/pages/game.vue:10]: I laughed when I saw this!

Comment (ZackWard-student) [reply wordle-ad-farm/pages/game.vue:10]: I honestly thought that you might start just playing adds and have no way out.

File: wordle-ad-farm/pages/game.vue [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<template>
+    
+    <v-card class='content-vue' >
+        
+        <v-card v-if="!isLoading" loading >
```
Comment (ZackWard-student) [inline wordle-ad-farm/pages/game.vue:5]: Love the loading effect!

**Review by ZackWard-student — APPROVED** [review]
Love the satire of the add machine! Oddly enough the original Wordle has no adds! From what I can tell you have met the requirements. If you follow the naming convention of the main dev branch merges should be less painful.

**Comment by tremorris1999** [comment]
@mmwoodfo See [DEPLOYMENT.md](https://github.com/tremorris1999/EWU-CSCD379-2022-Spring/blob/main/DEPLOYMENT.md) for deployed link. :)

**Comment by mmwoodfo** [comment]
In the future, just make sure that you are making a separate branch for your assignment, rather than making a PR using your main. Code changes showing up here in the PR should only be relevant to the assignment the PR is for.

Here is an example of a PR that uses an Assignment 1 branch: https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/pull/11

Great job!

**Comment by mmwoodfo** [comment]
> @mmwoodfo See [DEPLOYMENT.md](https://github.com/tremorris1999/EWU-CSCD379-2022-Spring/blob/main/DEPLOYMENT.md) for deployed link. :)

image
Very nice. :(

**Comment by mmwoodfo** [comment]
image
Sorry - this made me laugh.

Great job guys. :)

**Pull request closed without merging**